sql/ddl_api.sql

```diff
@@ -62,6 +62,25 @@ CREATE OR REPLACE FUNCTION @extschema@.create_distributed_hypertable(
     data_nodes              NAME[] = NULL
 ) RETURNS TABLE(hypertable_id INT, schema_name NAME, table_name NAME, created BOOL) AS '@MODULE_PATHNAME@', 'ts_hypertable_distributed_create' LANGUAGE C VOLATILE;
 
+-- A generalized hypertable creation API that can be used to convert a PostgreSQL table
+-- with TIME/SERIAL/BIGSERIAL columns to a hypertable.
+--
+-- relation - The OID of the table to be converted
+-- partition_column - Name of the partition column
```

Comment:
The comments for the arguments doesn't match the actual arguments used

Reply:
Fixed